packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2418,28 +2419,26 @@ export default class Meeting extends StatelessWebexPlugin {
       joinSuccess = resolve;
     });
 
-    // If Move or PSTN try merging into one
-    if (false) {
-      Metrics.postEvent({
-        event: eventType.CALL_INITIATED,
-        meeting: this,
-        data: {pstnAudioType: 'DIAL_IN'}
-      });
+    if (options.rejoin) {
+      LoggerProxy.logger.log(`Meeting:index#join --> Generating a new correlation id for meeting ${this.id}`);
+      LoggerProxy.logger.log(`Meeting:index#join --> Previous correlation id ${this.correlationId}`);
+      this.setCorrelationId(uuid.v4());
+      LoggerProxy.logger.log(`Meeting:index#join --> New correlation id ${this.correlationId}`);
+      this.meetingFiniteStateMachine.reset();
```

Comment:
Does this line changes the statemachine where it maintains IDLE -RINGING -JOINED stats !?

Reply:
Yes, since we haven't joined the meeting yet with a new correlation id, it will reset the state machine back to default for the meeting.